src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -164,6 +168,10 @@ public boolean hasAssessmentByName(AssessmentName name) {
         return assessments.containsByName(name);
     }
 
+    public Assessment getAssessmentByName(AssessmentName name) {
+        return assessments.getByName(name);
```

Comment:
Is it better to rename to getAssessmentWithName to standardise with the above changes for getPersonWithName!

Reply:
good idea! maybe i do the standardisation after we merge this pr?